tests/framework/builder.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Define some helpers methods to create microvms from artifacts."""
+
+import json
+import os
+import uuid
+import host_tools.cargo_build as build_tools
+from framework.microvm import Microvm
+from framework.artifacts import Artifact, DiskArtifact
+
+
+class MicrovmBuilder:
+    """Build fresh microvms or restore from snapshot."""
+
+    def __init__(self, root_path, bin_cloner_path):
+        """Initialize microvm root and cloning binary."""
+        self.root_path = root_path
+        self.bin_cloner_path = bin_cloner_path
+
+    def build(self, kernel: Artifact, disks: [DiskArtifact], config: Artifact):
+        """Build and start a fresh microvm."""
+        microvm_id = str(uuid.uuid4())
+        fc_binary, jailer_binary = build_tools.get_firecracker_binaries()
+
+        vm = Microvm(
+            resource_path=self.root_path,
+            fc_binary_path=fc_binary,
+            jailer_binary_path=jailer_binary,
+            microvm_id=microvm_id,
+            bin_cloner_path=self.bin_cloner_path
+        )
+        vm.setup()
```

Comment:
Unfortunately it doesn't work 😢 if I try to import it from conftest.py . This function will need to be changed in the future when we add firecracker binary artifacts, so indeed we are duplicating some code, with the promise that it will diverge at some point soon.

Reply:
Hmm, I just tried that now and it worked for me :-?.
```python
from conftest import init_microvm
...
vm = init_microvm(self.root_path, self.bin_cloner_path)
```
Isn't this working or am I doing something wrong?